cmd/gardenlet/app/migration.go

```diff
@@ -36,6 +44,43 @@ func (g *garden) runMigrations(ctx context.Context, log logr.Logger, _ cluster.C
 	return nil
 }
 
+// TODO(timuthy): Remove after v1.96 was released.
+// updateGardenerResourceManager updates all GRM deployments in a seed cluster,
+// in order to roll out the Brotli compression support added in Gardener v1.96.
+func updateGardenerResourceManager(ctx context.Context, log logr.Logger, cl client.Client) error {
+	image, err := imagevector.ImageVector().FindImage(imagevector.ImageNameGardenerResourceManager)
+	if err != nil {
+		return err
+	}
+	image.WithOptionalTag(version.Get().GitVersion)
+
+	grmDeployments := &appsv1.DeploymentList{}
+	if err := cl.List(ctx, grmDeployments, client.MatchingLabels(map[string]string{"app": "gardener-resource-manager"})); err != nil {
```

Comment:
Purely informational: Do you use the explicit string value for label key/value instead of constants to keep the dependencies small or are there other things to consider?

Usually, I would have voted for use of constants, but as this is more or less temporary code the literal approach is also fine with me.